content/en/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Observability Primer
+weight: 9
+---
+
+Before digging into OpenTelemetry, it is important to understand some core concepts first.
+
+## What is Observability?
+
+Observability lets us understand a system from the outside, by letting us ask questions about that system without knowing its inner workings. Furthermore, allows us to easily troubleshoot and handle novel problems (i.e. “unknown unknowns”), and helps us answer the question, "Why is this happening?" *without changing your code*.
```

Comment:
is this a definition ob observability everybody within the OTel community can "live with" (I can live with it, I am just highlighting that there are LOTS of definitions for that)

> *without changing your code*

For someone who is new to opentelemetry this part of the sentence might be confusing when you read the next one ("code must be properly instrumented"), maybe that "without chaning your code" needs some qualification

Reply:
Good point. I've updated the text to:

>Observability lets us understand a system from the outside, by letting us ask questions about that system without knowing its inner workings. Furthermore, allows us to easily troubleshoot and handle novel problems (i.e. “unknown unknowns”), and helps us answer the question, “Why is this happening?”
>
>In order to be able to ask those questions of a system, the application must be properly instrumented. That is, the application code must emit traces, metrics, and logs. An application is properly instrumented when developers don’t need to add more instrumentation to troubleshoot an issue, because they have all of the information they need.